### PR TITLE
Building with Active Record: Jr. Reddit!

### PR DESCRIPTION
**Pull request by lmaldonadoch**
On this project, we learned about Active Record, how to associate databases, and how to use the console to test them out.
## Project milestones
- Build tables using Rails built-in database interface.
- Create 3 different tables and associate them using Active Record.
- Create new records for each table and associate each one with its respective other.

# Requirements

## Section:
- Do not commit debugging instructions - read discussions about its reasons here.
- Keep your code clean.
- Remember to not change migration files after they are committed (explanation why it is important).
- Make sure that changes in migrations files are reflected in DB/schema.rb (explanation why it is important).
- Make sure that there is no logic in views and that helpers are responsible only for view related logic.
   - Where Do I Put My Code?
   - The Beginner’s Guide to Rails Helpers

## General requirements:
- [x] Add a descriptive README file to your repositories as described here.
- [x] Use meaningful names for your Pull Requests and add a short summary of its content as described here.
- [x] Use descriptive commit messages as described here.
- [x] Do not commit files that are not related to the project. Instead, add them to .gitignore file.
- [x] Do not commit inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history.

## Git Flow:
- [x] Use a specific branch for each feature.
- [x] Use a descriptive name for your branches.
- [x] Use Rubocop to validate your code.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  # validates :name, :username, :email
```
Comment (maelfosso) [inline app/models/user.rb]: Remove comments, please.

**Review by maelfosso — CHANGES_REQUESTED** [review]
# Changes required :red_circle: 

Hello @lmaldonadoch @shubham14p3 

Nice job done so far. However, you missed a requirement.
As shown [here](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/building_with_active_record/post_has_many_comments.png), a `Post` has many `Comments`. 

Kindly make the changes and submit a new code review request
Thanks.

Mael FOSSO

File: app/models/user.rb (hunk 1 above)
Comment (lmaldonadoch) [reply app/models/user.rb]: Hi! Thanks for pointing it out! We have fixed the issue by adding a has_many relationship to the post model. We added comments to Posts 1 and 2 (post_id = 1, post_id = 2). Post 2 has 2 comments so you can call comments.first or comments.last to retrieve them. 
Happy coding!

**Review by nikasulo — APPROVED** [review]
# Great work @lmaldonadoch and @shubham14p3 :medal_military: 
Please merge and keep moving forward! I wish you the very best and see you at the top!  :rocket: 

##### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can.

**Pull request merged**